frontend/src/Series/Details/SeriesDetails.css

```diff
@@ -27,22 +26,19 @@
 .headerContent {
   display: flex;
   padding: 30px;
+  gap: 35px;
   width: 100%;
   height: 100%;
   color: var(--white);
 }
 
 .poster {
   flex-shrink: 0;
-  margin-right: 35px;
   width: 250px;
   height: 368px;
 }
 
 .info {
-  display: flex;
-  flex-direction: column;
-  flex-grow: 1;
```

Comment:
Width is not maintained any more.

<img width="2056" alt="image" src="https://github.com/user-attachments/assets/9ab46185-b66e-4b86-9d37-94a5908fcd9d" />

Reverting this change fixes it.